docs/src/release_notes/v1.21.md

```diff
@@ -6,16 +6,79 @@ For a complete list of changes, please refer to the
 [commits](https://github.com/cloudnative-pg/cloudnative-pg/commits/release-1.21)
 on the release branch in GitHub.
 
-## Version 1.21.5
+## Version 1.21.6
 
-**Release date:** Apr 24, 2024
+**Release date:** Jun 12, 2024
```

Comment:
note to self: this may be a GH diff artifact, but seems the 1.21.5 has been deleted